pyproject.toml

```diff
@@ -35,6 +35,7 @@ python-dotenv = "^1.0.0"
 questionary = "^2.0.1"
 requests = "^2.31.0"
 jinja2 = "^3.1.3"
+pyjwt = "^2.8.0"
```

Comment:
This dependency should add in ​ [​tool​.​poetry​.​group​.​test_integration​.​dependencies​]. Run make lock after add it.